pages/linux/gopass.md

```diff
@@ -0,0 +1,35 @@
+# gopass
+
+> Standard Unix Password Manager for Teams. Written in Go.
+
+- Startup config:
+
+`gopass init`
+
+- Show all "keystores":
```

Comment:
Not sure the quotes are needed here - same with the below example.

Reply:
I wasn't aware of the official name for it. Keystore, store, safe...
It seems it is "store". I will update it accordingly.